messages/ownship/steering_unit.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Steering unit",
+  "description": "Description over vessel model steering unit",
+  "required": ["unit_id"],
+  "properties": {
+    "unit_id": {
+      "type": "string",
+      "title": "Unit id",
+      "desciption": "Id of the reporting physical/virtual sensor equipment"
+    },
+    "location": {
+      "description": "Location of the rudder in the BF frame of reference.",
+      "$ref": "../core/location.json"
+    },
+    "readable_id": {
+        "type": "string",
+        "title": "Readable id",
+        "desciption": "Human readable identifier "
+    },
```

Comment:
What would be the difference between the `unit_id` and the `readable_id` ? Both have type `string` and there is nothing stopping us from having a human readable `unit_id` as far as I can tell?